### ISSUE TITLE
uvfits key describing the number of timesteps

### ISSUE THREAD

**Issue by cjordan** [MWATelescope/Marlu#2]
Not sure if there's an official uvfits key or not, but it's quite painful discovering the number of timesteps in a uvfits file -- one more or less has to read the entire file to determine it.  A key (even a non-standard one) would be very useful.  I guess using this key would also assume that the layout of the uvfits is regular (i.e. one timestep has x rows/baselines before the next timestep which also has x rows, etc.).

**Comment by d3v-null** [comment]
Yeah, I would probably call this `NSCANS` to be consistent with metafits and mwaf

**Comment by cjordan** [comment]
I just realised that this doesn't reveal what times are actually available in a uvfits file. What do you think about providing unique timestamps with some key too?
